spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -35,7 +35,7 @@ def handle_two_step(response)
         result = choose(*available)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
-      elsif r.body.kind_of?(Hash) && r.body["phoneNumberVerification"].kind_of?(Hash)
+      elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Hash)
```

Comment:
@meyskens thx for the feedback, i did a commit to handle the array, can you re-test it?